specification/compute/resource-manager/Microsoft.Compute/stable/2020-05-01/disk.json

```diff
@@ -1391,6 +1732,33 @@
     },
     "EncryptionSetProperties": {
       "properties": {
+        "encryptionType": {
+          "type": "string",
+          "description": "The type of key used to encrypt the data of the disk. EncryptionType can't be changed once the disk encryption set has been created",
+          "enum": [
+            "EncryptionAtRestWithPlatformKey",
+            "EncryptionAtRestWithCustomerKey",
+            "EncryptionAtRestWithPlatformAndCustomerKeys"
+          ],
+          "x-ms-enum": {
+            "name": "EncryptionType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "EncryptionAtRestWithPlatformKey",
+                "description": "Disk is encrypted with XStore managed key at rest. It is the default encryption type."
+              },
+              {
+                "value": "EncryptionAtRestWithCustomerKey",
+                "description": "Disk is encrypted with Customer managed key at rest."
```

Comment:
Encryption with a customer managed key that can be changed and revoked by a customer,